### PR TITLE
Fix errors | iQIYI

### PR DESCRIPTION
**Pull request by SlowLife1661**
;-;

### CODE REVIEW

**Review by Bas950 — DISMISSED** [review]

**Review by ririxidev — CHANGES_REQUESTED** [review]
isn't fixed. 
image

**Review by Bas950 — DISMISSED** [review]

File: websites/G/Google Messages/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-    "extends": "../../../tsconfig.json",
-    "compilerOptions": {
-      "outDir": "./dist/"
-    }
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
   }
```
Comment (ririxidev) [inline websites/G/Google Messages/tsconfig.json:6]: why you edit another presence tsconfig? xD

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/G/Google Messages/tsconfig.json (hunk 1 above)
Comment (SlowLife1661) [reply websites/G/Google Messages/tsconfig.json:6]: Not sure why it was edited 😅

**Review by Bas950 — DISMISSED** [review]

**Comment by SlowLife1661** [comment]
Will reopen this later

File: websites/G/Google Messages/tsconfig.json (hunk 1 above)
Comment (SlowLife1661) [reply websites/G/Google Messages/tsconfig.json:6]: Seems like I'm just dumb I don't know how to undo this plus the file looks just same

**Comment by ririxidev** [comment]
You already borked it enough. Just re-fork our presence repo and re-do these commits to fix errors in iQIYI.

**Comment by SlowLife1661** [comment]
e.e

**Pull request closed without merging**